libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -29,28 +29,27 @@ data class NavigationOptions constructor(
     val navigatorPollingDelay: Long,
     val distanceFormatter: DistanceFormatter?,
     val onboardRouterConfig: MapboxOnboardRouterConfig?,
-    val isFromNavigationUi: Boolean = false
+    val isFromNavigationUi: Boolean
 ) {
 
     /**
      * Get a builder to customize a subset of current options.
      */
-    fun toBuilder() = Builder(
-        roundingIncrement,
-        timeFormatType,
-        navigatorPollingDelay,
-        distanceFormatter,
-        onboardRouterConfig
-    )
+    fun toBuilder() = Builder()
+        .roundingIncrement(roundingIncrement)
+        .timeFormatType(timeFormatType)
+        .navigatorPollingDelay(navigatorPollingDelay)
+        .distanceFormatter(distanceFormatter)
+        .onboardRouterConfig(onboardRouterConfig)
+        .isFromNavigationUi(isFromNavigationUi)
 
-    data class Builder(
-        private var roundingIncrement: Int = ROUNDING_INCREMENT_FIFTY,
-        private var timeFormatType: Int = NONE_SPECIFIED,
-        private var navigatorPollingDelay: Long = DEFAULT_NAVIGATOR_POLLING_DELAY,
-        private var distanceFormatter: DistanceFormatter? = null,
-        private var onboardRouterConfig: MapboxOnboardRouterConfig? = null,
+    class Builder {
```

Comment:
Note that we can already do this because they all have default values. But this change will make it a little more clear that it's on purpose

https://github.com/mapbox/mapbox-navigation-android/blob/5a47bfa517f33f57fcc8cb0503c471174b91497a/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L556-L567